src/pages/benchmarks/index.tsx

```diff
@@ -16,12 +16,12 @@ function paraDashboard() {
       {tasks?.map((task) => {
         return (
           <li key={task.task_id}>
-            <TaskCard task={task} />
+            <TaskCard task={task as ParaTaskCard} />
```

Comment:
curious why the cast is needed?

Reply:
I believe I initially added this when the data was incomplete, but it's no longer necessary. I'll push up a couple edits.